index.js

```diff
@@ -53,7 +56,27 @@ bot.Dispatcher.on(Events.MESSAGE_CREATE, (c) => {
           if (Commands[cmd].phantom !== undefined) msg.reply('this command is restricted, and not available to you.')
           return
         } else if (level !== 2 && Commands[cmd].adminOnly === true) return
+        if (limits.global[cmd] && Date.now() - commandLimitMap.get('global') < limits.global[cmd] && level !== 2) {
```

Comment:
`commandLimitMap.get(msg.author.id+cmd)` is not guaranteed to return a value, but this is assumig it is.